bin/people_array.rb

```diff
@@ -4,3 +4,33 @@
 people = People.new.people
```

Comment:
@gaand, No it should not have been deleted (and I didn't think it was? the pluperfect is confusing me).

No, @MicFin not quite. The file you are commenting on is `bin/people_array.rb`. The file in the challenge is `lib/person.rb`.

This file is used during a demo. https://github.com/ga-wdi-boston/ruby-enumerable-io/tree/016/master#demo---csv-class